src/components/Menu/MenuItem.tsx

```diff
@@ -95,6 +106,10 @@ class MenuItem extends UIComponent<any, any> {
             onClick={this.handleClick}
             {...accessibility.attributes.anchor}
           >
+            {this.props.icon &&
+              Icon.create(this.props.icon, {
+                defaultProps: { xSpacing: !!content ? 'after' : 'none' },
+              })}
             {content}
```

Comment:
Should we consider using the Layout component for aligning the content inside the MenuItem? Are there going to be other types of menus with media in the start and end?

Reply:
Good point. I would suggest leaving it as it is for now and refactor by adding `Layout` later as new requirements for the component come.

Reply:
Sure!